docs/edit_first_game_server.md

```diff
@@ -33,20 +37,31 @@ go get agones.dev/agones/pkg/sdk
 GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o bin/server -a -v main.go
 ```
 
-## Using Docker File without using Minikube
+## Using Docker File
 
 ### Create a new docker image
 ```bash
 docker build -t gcr.io/[PROJECT_ID]/agones-udp-server:0.2 .
 ```
 Note that: you can change the image name "agones-udp-server" to something else.
 
-### Push the image to GCP Registry
+### If using GKE, push the image to GCP Registry
 ```bash
 docker push gcr.io/[PROJECT_ID]/agones-udp-server:0.2
```

Comment:
Wondering if we should link to:
https://cloud.google.com/container-registry/docs/advanced-authentication

To point people in the right direction to authenticate against the GCR?

Reply:
Do you mean add content along the lines of "More information is available at <link> "?

Reply:
Committed some text adding this link.

Reply:
Perfect :+1: